### DISCUSSION TITLE
"本地 IPv4 绕过地址"功能失效

### DISCUSSION THREAD

**Discussion by xiaoyangdkj** [vernesong/OpenClash#3862] [Q&A] [answered]
在使用meta核的状态下，无论是兼容模式还是fake-ip模式，设置的绕过名单均不会生效，流量仍经过内核
![Screenshot_20240429_091711_com.android.chrome_edit_96165031823345.jpg](https://github.com/vernesong/OpenClash/assets/82305021/d121df24-c738-4d8e-8581-c2989160c496)

![Screenshot_20240429_092122_com.android.chrome_edit_96131226328559.jpg](https://github.com/vernesong/OpenClash/assets/82305021/6d6d6b44-fd89-4932-9961-82e34122eca6)

**Comment by Vegas7** [comment]
我也发现这个问题，但有些域名又可以生效，如chiphell.com。

**Reply by Vegas7** [reply]
似乎是80端口的不生效

**Comment by slackworker** [comment, marked as answer]
不知道你想实现什么策略，但是看起来你对这个功能有误解
1、这里设置的是目标ip地址，直接添加到fw3/fw4不进入clash核心
2、fw3/fw4中格式必须是ip，不能是域名
